pages/requests/create-renewal.tsx

```diff
@@ -0,0 +1,208 @@
+        <GridItem paddingTop="32px">
+          <Box
+            border="1px solid"
+            borderColor="border.secondary"
+            borderRadius="12px"
+            bgColor="white"
+            paddingTop="32px"
+            paddingBottom="40px"
+            paddingX="40px"
+            align="left"
+          >
+            <Text textStyle="display-small-semibold" paddingBottom="20px">
+              {`Doctor's Information`}
+            </Text>
+            <DoctorInformationForm
+              doctorInformation={doctorInformation}
+              onChange={setDoctorInformation}
+            />
+          </Box>
+        </GridItem>
+        {/* Additional Quesitons Form */}
+        <GridItem paddingTop="32px">
+          <Box
+            border="1px solid"
+            borderColor="border.secondary"
+            borderRadius="12px"
+            bgColor="white"
+            paddingTop="32px"
+            paddingBottom="40px"
+            paddingX="40px"
+            align="left"
+          >
+            <Text textStyle="display-small-semibold" paddingBottom="20px">
+              {`Additional Information`}
+            </Text>
+            <AdditionalQuestionsForm data={additionalQuestions} onChange={setAdditionalQuestions} />
+          </Box>
+        </GridItem>
+        {/* Payment Details Form */}
+        <GridItem paddingTop="32px" paddingBottom="68px">
+          <Box
+            border="1px solid"
+            borderColor="border.secondary"
+            borderRadius="12px"
+            bgColor="white"
+            paddingTop="32px"
+            paddingBottom="40px"
+            paddingX="40px"
+            align="left"
+          >
+            <Text textStyle="display-small-semibold" paddingBottom="20px">
+              {`Payment, Shipping, and Billing Information`}
+            </Text>
+            <PaymentDetailsForm paymentInformation={paymentDetails} onChange={setPaymentDetails} />
+          </Box>
+        </GridItem>
+        {/* Footer */}
+        <Box
+          position="fixed"
+          left="0"
+          bottom="0"
+          right="0"
+          paddingY="20px"
+          paddingX="188px"
+          bgColor="white"
+          boxShadow="dark-lg"
+        >
+          <Stack direction="row" justifyContent="space-between" alignItems="center">
+            <Box>
+              <Text textStyle="body-bold">
+                User ID:
+                <Box as="span" color="primary">
+                  {' '}
+                  {permitHolderID}
+                </Box>
+              </Text>
+            </Box>
+            <Box>
+              <Button
+                bg="background.gray"
+                _hover={{ bg: 'background.grayHover' }}
+                color="black"
+                marginRight="20px"
+                height="48px"
+                width="149px"
+              >
+                <Text textStyle="button-semibold">Cancel</Text>
+              </Button>
```

Comment:
Wrap the `Button` with a Next.js `Link` component that links to `/admin` (requests table page)